README.md

```diff
@@ -115,55 +115,49 @@ Each Oak Module must expose the following **exported functions** as
     may fail to be invoked). No further interactions with the Oak Node are
     possible after finalization.
 
--   `oak_handle_grpc_call: () -> nil`: Invoked when a client interacts with the
-    Oak Node over gRPC. Each client interaction results in a new invocation of
-    this function, and the Oak VM guarantees that concurrent invocations only
+-   `oak_handle_grpc_call: (i64) -> nil`: Invoked when a client interacts with
+    the Oak Node over gRPC. Each client interaction results in a new invocation
+    of this function, and the Oak VM guarantees that concurrent invocations only
     invoke it sequentially, therefore from the point of view of the Oak Node
     these calls will never overlap in time and each execution of this function
     has full access to the underlying internal state until it completes. In the
     future we may relax some of these restrictions, when we can reason more
     accurately about the semantics of concurrent invocations, and how they
     relate to the policy system.
 
+    *   arg 0: Id of the built-in gRPC channel, used to read the incoming gRPC
+        request and write the outgoing gRPC response; this channel is implicitly
+        available to any invocation and can be used with the standard methods
+        below, but it cannot be opened or closed.
+
 Each Oak Module may also optionally rely on zero or more of the following **host
 functions** as
 [WebAssembly imports](https://webassembly.github.io/spec/core/syntax/modules.html#imports)
 (all of them defined in the `oak` module):
 
--   `print: (i32, i32) -> nil`: Prints a string to standard output on the host
-    system. To be used for debugging only, as it leaks data. Will be removed
-    before release.
-
-    *   arg 0: Source buffer address
-    *   arg 1: Source buffer size in bytes
+-   `open_channel (i32, i32) -> i64`: Opens the channel specified by the
```

Comment:
The description above says the channel cannot be opened or closed.
Should this be "create"?  Does it need to be explicitly created?

Reply:
No, this host function is for opening channels in general, just not the `grpc` channel. Similar to how you cannot "open" stdin or stdout, they are just already there. Does that make sense?